src/foundry/prosemirror/util.d.ts

```diff
@@ -0,0 +1,31 @@
+import type { Node, Schema } from 'prosemirror-model';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import type { StringNode } from './string-serializer';
```

Comment:
What is this for?

Reply:
Brought in strictly to make VS Code happy rendering `{@link StringNode#toString}` right as a link, as it's not a global class instance. Do you know another way to do that?

Reply:
Oh weird I didn't know VSC needed that. In that case we have a nice tooling conflict here. It's fine to leave it in, but can you please add a comment why this is done? Then it's good from my side.

Reply:
Comment added.